### ISSUE TITLE
No Markdown editor available

### ISSUE THREAD

**Issue by Zalfsten** [fairagro/basic_infrastructure#68]
As we're facing issue with too few OnlyOffice licences/sessions, people started to switch to Markdown. So it would be nice to have a good collaborative Markdown editor for Nextcloud. Unfortunately it's not available. The [Markdown Editor](https://apps.nextcloud.com/apps/files_markdown) is not compatible to Netxcloud 28. There are a bunch of open issues that are not worked on, e.g.: https://github.com/nextcloud/files_texteditor/issues/816.
Another Markdown editor [HedgeNext](https://apps.nextcloud.com/apps/hedgenext) does not work with our Nextcloud installation. The app won't activate with nextcloud saying it may have issues on the server stability. But it is expected to be compatible with Nextcloud 28 (our version) and 29.

How to proceed? We cannot downgrade to Nextcloud 27 to be able to use the Markdown editor, because this version is out of support.

Maybe we could try to find out, why HedgeNext is not working for us.

**Comment by darnold-zalf** [comment]
Unfortunately, the only 2 options I realistically see for this are:

1. Purchase more OnlyOffice licenses, or:
2. Wait on files_markdown (official NC markdown-editor) to be released with a compatible NC28 version.

Note that downgrading NC from v28 to v27 is not possible

- files_markdown
    - https://github.com/icewind1991/files_markdown 
    - The official (inbuilt) nextcloud markdown-editor application
    - It seems NC28 introduced alot of breaking changes (We are at NC version 28)
    - developer doesnt have time to fix the NC28 breaking changes
	 - https://github.com/nextcloud/files_texteditor/issues/816#issuecomment-2050019810
	 - https://github.com/icewind1991/files_markdown/issues/218#issuecomment-2049982250
- hedgedoc
    - Independent, seperate markdown-editor, it has no affiliation/relationship with Nextcloud.
    - It does support NC authentication, but nothing else ..
        - https://docs.hedgedoc.org/guides/auth/nextcloud/
    - There were 2 forks of this project (below), but unfortunately both forks seem abandoned.
- hedgenext-plugin
    - https://github.com/libnewton/hedgenext-plugin
    - This was a fork of hedgedoc to get it integrated with NC.
    - There _was_ a release to bump to NC29, but it seems not fully validated/working
    - project is in maintenance mode
- hedgenext-server
    - https://github.com/libnewton/hedgenext-server
    - This was a fork of hedgedoc to get it integrated with NC.
    - Seems abandonded, no changes since Feb2023.

**Comment by Zalfsten** [comment]
Ok, so my assumption is that the hedgenext-plugin could work. But it requires a Hedgedoc server to talk to. That sounds like a bigger task.

**Comment by darnold-zalf** [comment]
Possible workaround here : https://github.com/icewind1991/files_markdown/issues/218#issuecomment-2565425000
